UNUMColorPicker/Classes/UNUMColorPickerViewController.swift

```diff
@@ -42,6 +42,11 @@ public class UNUMColorPickerViewController: UIViewController {
         self.viewModel = UNUMColorPickerViewModel(colors: colors, initialColor: initialColor)
     }
 
+    public func reset(initialColor color: UIColor) {
+        self.viewModel.initialColor = color
+        colorCollectionView.reloadData()
+    }
```

Comment:
not sure i understand the use of this...

Reply:
we can use this when we want to change other page's background color, so we don't need to create a new color picker view, but just reset the initial color of the existing color picker view